rewrite-core/src/main/java/org/openrewrite/trait/Reference.java

```diff
@@ -52,11 +52,13 @@ interface Provider {
 
     interface Matcher {
         boolean matchesReference(Reference value);
+        default Renamer renameTo(String newName) {
```

Comment:
This method return a new `Renamer` but does not rename things. I this a better name would be `getRenamer` or `getReferenceRenamer`.